pypsa/plot.py

```diff
@@ -254,11 +267,103 @@ def as_branch_series(ser):
     ax.update_datalim(compute_bbox_with_margins(margin, x, y))
     ax.autoscale_view()
 
+    if basemap_present:
+        ax.axis('off')
+    if cartopy_present and geomap:
+        ax.outline_patch.set_visible(False)
+
     ax.set_title(title)
 
     return (bus_collection,) + tuple(branch_collections)
 
 
+def get_projection_from_crs(crs):
+    if crs == 4326:
+        # if data is in latlon system, return default map with latlon system
+        return ccrs.PlateCarree()
+    try:
+        return ccrs.epsg(crs)
+    except ValueError:
```

Comment:
`ccrs.epsg` requires an internet exception ([doc](https://scitools.org.uk/cartopy/docs/latest/crs/index.html#cartopy.crs.epsg)). If the request fails, the package 'pyepgs', used by cartopy, [will raise a `requests.RequestException`](https://github.com/rhattersley/pyepsg/blob/master/pyepsg.py#L287). 
```suggestion
    except requests.RequestException:
        logger.warning("A connection to http://epsg.io/ is required for a projected coordinate reference system. "
                    "Falling back to latlong.")
    except ValueError:
```
(The `requests` package should be imported.)